avaframe/com1DFA/com1DFA.py

```diff
@@ -1628,19 +1631,27 @@ def initializeResistance(cfg, dem, simTypeActual, resLine, reportAreaInfo, thres
     """
     K = cfg.getfloat("detK")
     detrainment = cfg.getboolean("detrainment")
-    detWithoutRes = cfg.getboolean("detWithoutRes")
 
-    ResModel = cfg["ResistanceModel"].lower()
-    if ResModel == "cres":
-        cRes = cfg.getfloat("cRes")
-    if ResModel == "cresh":
-        cRes = cfg.getfloat("cResH")
     # read dem header
     header = dem["originalHeader"]
     ncols = header["ncols"]
     nrows = header["nrows"]
 
     if simTypeActual in ["entres", "res"]:
+        ResModel = cfg["ResistanceModel"].lower()
+        if ResModel == "default":
+            cRes = cfg.getfloat("cResH")
+            if detrainment:
+                log.info('Resistance model default: if FT < %.2f OR FV < %.2f detrainment (detK %.2f) else increased friction (cResH %.2f) and no effect if FT > %.2f OR FV > %.2f' %
+                         (cfg.getfloat('forestThMin'), cfg.getfloat('forestVMin'), K, cfg.getfloat('cResH'),
+                          cfg.getfloat('forestThMax'), cfg.getfloat('forestVMax')))
+            else:
+                log.warning('Resistance model default chosen, but detrainment is deactivated apply increased friction (cResH %.2f) everywhere within resistance area' % cfg.getfloat('cResH'))
```

Comment:
I don't think this is necessary, I suggest removing it

Reply:
I find this a quite helpful information in the log

Reply:
If you want to keep it, then please drop it to debug. In my opinion this is unnecessary, since it is described in the documentation as well as written in the ini file and additionally these values are also written in the log earlier on. And if someone changes the detrainment  in the ini file, they are aware of it...
If we start writing the configuration setup as separate log messages as well, then it's going to be a very messy log....